index.js

```diff
@@ -35,8 +35,13 @@ var gulpSass = function gulpSass(options, sync) {
 
 
     opts = clonedeep(options || {});
-    opts.data = file.contents.toString();
-
+    if (opts.data === void 0) {
+      opts.data = file.contents.toString();
+    }
+    else {
+      opts.data = opts.data + file.contents.toString();
```

Comment:
Please see my comments on https://github.com/dlmanning/gulp-sass/issues/599#issuecomment-372105425 and #302 for why I don't think this change is correct.